podman/domain/containers_create.py

```diff
@@ -403,6 +404,9 @@ def to_bytes(size: Union[int, str, None]) -> Union[int, None]:
                     f"or int (found : {size_type})"
                 )
 
+        if "working_dir" in args:
+            warnings.warn("working_dir is deprecated, use workdir instead", PendingDeprecationWarning)
+
```

Comment:
Please remove the warning. The library is intended to be a super set of the Docker SDK which uses `working_dir` as their parameter. I have no issue with workdir being added as an "alias".